File: src/main/java/no/nav/tjenestepensjon/simulering/consumer/TpRegisterConsumer.java
package no.nav.tjenestepensjon.simulering.consumer;

import java.util.List;

import no.nav.tjenestepensjon.simulering.domain.TPOrdning;
import no.nav.tjenestepensjon.simulering.exceptions.NoTpOrdningerFoundException;

public interface TpRegisterConsumer {

    List<TPOrdning> getTpOrdningerForPerson(String fnr) throws NoTpOrdningerFoundException;
}

File: src/main/kotlin/no/nav/tjenestepensjon/simulering/consumer/TpRegisterConsumer.kt
package no.nav.tjenestepensjon.simulering.consumer

import no.nav.tjenestepensjon.simulering.model.domain.TPOrdning
import no.nav.tjenestepensjon.simulering.exceptions.NoTpOrdningerFoundException
import no.nav.tjenestepensjon.simulering.model.domain.FNR

interface TpRegisterConsumer {
    @Throws(NoTpOrdningerFoundException::class)
    fun getTpOrdningerForPerson(fnr: FNR): List<TPOrdning>
}